frontend/src/App.test.tsx

```diff
@@ -25,13 +25,15 @@ import {
   Config,
   CustomThemeConfig,
   ForwardMsg,
+  ForwardMsgMetadata,
   ICustomThemeConfig,
   INewSession,
   NewSession,
   PageConfig,
   PageInfo,
   PageNotFound,
   PagesChanged,
+  Delta,
```

Comment:
supernit: Should this go to line 26 so it's alphabetical 😆

Reply:
Looking at other files though, it doesn't seem like we retain that so I don't think it's necessary. Just something I noticed... 😅

Reply:
Yeah - this is the sort of thing that we should enforce with linters if we care, because it's purely stylistic.